src/components/EditCoursePage/index.jsx

```diff
@@ -448,6 +450,7 @@ class EditCoursePage extends React.Component {
       !IN_REVIEW_STATUS.includes(targetRun.status)) {
       // Submitting Run for review, show modal, and temporarily store form data until
       // we have a response for how to continue.
+      store.dispatch(courseRunSubmitting());
```

Comment:
I honestly agree with this, but the current pattern right now is to pass in actions as props from the container.  idk i honestly think this is so much easier to read, but its not done like this in our project very often